recipes/heatshrink/all/CMakeLists.txt

```
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.4)
+project(heatshrink C)
+
+if (WIN32 AND BUILD_SHARED_LIBS)
+    set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
+endif(WIN32 AND BUILD_SHARED_LIBS)
+
+SET(SOURCE_DIR ${CMAKE_SOURCE_DIR}/${SOURCE_SUBDIR} )
```

You can hardcode `source_subfolder` here and don't bother with passing it in `build` of the conan recipe.
```suggestion
SET(SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/source_subfolder")
```